sycl/source/detail/queue_impl.hpp

```diff
@@ -850,12 +855,12 @@ class queue_impl {
     auto EventRet = Handler.finalize();
     EventImplPtr EventRetImpl = getSyclObjImpl(EventRet);
     if (Type == CGType::CodeplayHostTask)
-      Deps.UnenqueuedCmdEvents.push_back(EventRetImpl);
+      Deps.UnenqueuedCmdEvents.push_back(std::move(EventRetImpl));
```

Comment:
`getSyclObjImpl(EventRet)` vs `std::move(EventRetImpl)` is almost equal, I don't think the temporary variable is justified anymore. IMO, inline it and maybe add a comment for the future as well.

Reply:
Done.

Reply:
It's turn out this was already modified on sycl branch (reference used instead). So, dropped from the patch.